docs/src/api/websocket/_slotsUpdatesSubscribe.mdx

```diff
@@ -55,7 +55,7 @@ This subscription is unstable
 
 The notification will be an object with the following fields:
 
-- `parent: <u64>` - The parent slot
+- `parent: <u64|undefined>` - The parent slot. Only present if the update is of type "createdBank".
```

Comment:
Looks right per
https://github.com/solana-labs/solana/blob/21b2fce6f47899eb3f8d6be1f18431c808349811/rpc-client-api/src/response.rs#L224-L228

If you dig around in that enum in Rust, you can see there are some other fields for specific responses. Namely,
https://github.com/solana-labs/solana/blob/21b2fce6f47899eb3f8d6be1f18431c808349811/rpc-client-api/src/response.rs#L229-L232
and
https://github.com/solana-labs/solana/blob/21b2fce6f47899eb3f8d6be1f18431c808349811/rpc-client-api/src/response.rs#L234-L237
both have an extra field as well

Reply:
Cool, added these!